osquery/config/config.cpp

```diff
@@ -530,6 +541,54 @@ void stripConfigComments(std::string& json) {
   json = sink;
 }
 
+std::map<std::string, std::string> Config::restoreConfigBackup() {
```

Comment:
Can this method fail? If so, would you like to use Expected for return value?

Reply:
Done. thank